src/3rdparty/walkontable/src/overlays.js

```diff
@@ -392,8 +391,10 @@ class Overlays {
 
     const [scrollLeft, scrollTop] = [this.scrollableElement.scrollLeft, this.scrollableElement.scrollTop];
 
-    this.horizontalScrolling = topHolder.scrollLeft !== scrollLeft;
-    this.verticalScrolling = leftHolder.scrollTop !== scrollTop;
+    this.horizontalScrolling = (topHolder.scrollLeft !== scrollLeft || this.lastScrollX !== window.scrollX);
+    this.verticalScrolling = (leftHolder.scrollTop !== scrollTop || this.lastScrollY !== window.scrollY);
+    this.lastScrollX = window.scrollX;
+    this.lastScrollY = window.scrollY;
```

Comment:
Please add:
```
this.lastScrollX = window.scrollX;
this.lastScrollY = window.scrollY;
```
to the `constructor() {...}`. It will resolve problems with tests.